Tasks/KubernetesManifestV1/src/utils/DeploymentHelper.ts

```diff
@@ -34,7 +34,14 @@ export async function deploy(kubectl: Kubectl, manifestFilePaths: string[], depl
 
     // check manifest stability
     const resourceTypes: Resource[] = KubernetesObjectUtility.getResources(deployedManifestFiles, models.deploymentTypes.concat([KubernetesConstants.DiscoveryAndLoadBalancerResource.service]));
-    await checkManifestStability(kubectl, resourceTypes);
+    const resourceType = tl.getInput('resourceType') || ''
+
+    // for a fleet object, we do not check for manifest stability
+    if (resourceType.toLowerCase() != 'microsoft.containerservice/fleets') {
+        await checkManifestStability(kubectl, resourceTypes)
+    }else{
+        tl.debug('checkManifestStability was skipped for fleet');
+    }
```

Comment:
Nit: not sure if we need the else just to log things (it's non-functional and will be harder to write coverage test).  Might be a tad better to just log in the if clause and get rid of else completely.